app/styles/_tables.less

```diff
@@ -143,6 +143,19 @@
         }
       }
     }
+    // remove extra 2px border on <tr> in nested ng-repeats.
+    // since we cannot select the "first element of parent with class ..."
+    // select every <tr> with the .nested-loop class, apply a 0px border-top-width,
+    // and then re-apply the 2px border-top-width to every tr.nested-loop child
+    // except the first one to avoid tables with 4px border-top-widths.
+    > tbody.nested-loop-container {
+      > tr.nested-loop {
+        border-top-width: 0;
+      }
+      > tr.nested-loop ~ tr.nested-loop {
+          border-top-width: 2px;
```

Comment:
@sg00dwin @jwforres @spadgett 
This works for all cases where there must be sibling elements (with the first one set to `display: none;`) in order to have nested loops (`traffic-table.html` and `deployments.html` in this PR). I first remove the border from all `<tr>` elements in the loop, and then re-apply it to every `<tr>` in that loop except the first one with `tr.nested-loop ~ tr.nested-loop`.